WTWSMS/common/religions/00_religions.txt

```diff
@@ -4337,7 +4330,6 @@ pagan_group = {
 		max_consorts = 3
 		
 		ai_convert_same_group = 0 # never
-		ai_convert_other_group = 1 # only when zealous

```

Comment:
It should be `ai_convert_other_group = 1 # only when zealous` rather than `ai_convert_other_group = 2 # always try to convert`.